ricecooker/chefs.py

```diff
@@ -281,10 +282,49 @@ def save_channel_tree_as_json(self, channel):
         self.CHEF_RUN_DATA['tree_archives']['current'] = filename.replace(os.getcwd() + '/', '')
         self.save_chef_data()
 
+    def save_channel_metadata_as_csv(self, channel):
+        # create data folder in chefdata
+        # config.LOGGER.info('SAVE METADATA TO CSV')
+        DATA_DIR = os.path.join('chefdata', 'data')
+        if not os.path.isdir(DATA_DIR):
+            os.makedirs(DATA_DIR)
+        metadata_csv = csv.writer(open(os.path.join(DATA_DIR, 'content_metadata.csv'), 'w', newline=''))
+        headers = [
+            'Source_id',
```

Comment:
Can we call this "Source ID" to be consistent with the formatting of the other field names?

Reply:
Also, it may be a good idea to create constants for these in `ricecooker.config`, and use those constants here instead of that actual string. That way, if we want to change the strings, we won't have to search the code for the string.